src/components/app/details/appDetails/DeploymentStatusCard.tsx

```diff
@@ -83,12 +87,14 @@ function DeploymentStatusCard({
         if (!hideDetails) {
             refetchDeploymentStatus(true)
         }
-        if (loadingResourceTree) noop()
+        if (cardLoading) noop()
```

Comment:
We can remove this line

Reply:
Removing this directly in this pr then : https://github.com/devtron-labs/dashboard/pull/1482/files#r1399372332